docs/rbe/rbe.md

```diff
@@ -116,6 +157,10 @@ builds.
 
 The status of the local RBE proxy can be queried with the following command
 
+```
+buildtools/mac-arm64/reclient/reclientreport
```

Comment:
This file doesn't seem to exist on my machine.

Reply:
Hmm. Do you see it under `//flutter/buildtools/...` ?

Reply:
Ah, wrong buildtools. Yeah, I see it there. But that just seems to generate a tarball with logs that are pretty hard to parse. Not sure we should include it as a debugging aid.

Reply:
Whoops. It should have been `reproxystatus` not `reclientreport`.

Reply:
That always reports that reproxy is not running. Presumably because et starts and stops it as needed?